frontend/src/__tests__/basic.test.js

```diff
@@ -0,0 +1,163 @@
+ *
+ * @param {object} api
+ * @param {Function} api.apiGET A function that when passed a route will return the get response
+ * @param {Function} api.apiPOST A function that when passed a route and data, will return the post response
+ */
+function minimalSetupTest(api = { apiGET, apiPOST }) {
+    const { apiGET, apiPOST } = api;
+    let session = null;
+
+    it("create a session", async () => {
+        await apiPOST("/admin/debug/clear_data");
+
+        // create session
+        const newSessionData = {
+            start_date: new Date("2019/09/09").toISOString(),
+            end_date: new Date("2019/12/31").toISOString(),
+            name: "First Session",
+            rate1: 56.54
+        };
+
+        const { payload: createdSession } = await apiPOST(
+            "/admin/sessions",
+            newSessionData
+        );
+        const createdSessionId = createdSession.id;
+
+        // create contract template
+        const newContractTemplateData = {
+            template_name: "standard",
+            template_file: "/math/default.html"
+        };
+
+        const { payload: createdContractTemplate } = await apiPOST(
+            `/admin/sessions/${createdSessionId}/contract_templates`,
+            newContractTemplateData
+        );
+        const createdContractTemplateId = createdContractTemplate.id;
+
+        // create position
+        const newPositionData = {
+            contract_template_id: createdContractTemplateId,
+            position_code: "MAT135F",
+            position_title: "Calculus I",
+            hours_per_assignment: 70,
+            start_date: new Date("2019/09/09").toISOString(),
+            end_date: new Date("2019/12/31").toISOString()
+        };
+
+        const { payload: createdPosition } = await apiPOST(
+            `/admin/sessions/${createdSessionId}/positions`,
+            newPositionData
+        );
+        const createdPositionId = createdPosition.id;
+
+        // create applicant
+
+        const newApplicantData = {
+            utorid: "weasleyr",
+            student_number: "89013443",
+            first_name: "Ron",
+            last_name: "Weasley",
+            email: "ron@potter.com",
+            phone: "543-223-9993"
+        };
+
+        const { payload: createdApplicant } = await apiPOST(
+            `/admin/applicants`,
+            newApplicantData
+        );
+        const createdApplicantId = createdApplicant.id;
+
+        // create assignment
+        const newAssignmentData = {
+            position_id: createdPositionId,
+            applicant_id: createdApplicantId,
+            start_date: new Date("2019/09/09").toISOString(),
+            end_date: new Date("2019/12/31").toISOString(),
+            hours: 90
+        };
+
+        const { payload: createdAssignment } = await apiPOST(
+            `/admin/assignments`,
+            newAssignmentData
+        );
+
+        // fetch all the data and ensure the newly created data comes back
```

Comment:
Make this comment stand out more.

Reply:
Not sure if I understand correctly, but I have use the comment block for this part, and at line 29  since they are comments about the general idea